taskcluster/worker_images_taskgraph/util/fxci.py

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+from collections import defaultdict
+from functools import cache
+
+import taskcluster
+
+FIREFOXCI_ROOT_URL = "https://firefox-ci-tc.services.mozilla.com"
```

Comment:
nit: Should this be hardcoded? In decision tasks it will already be set to the correct value in the env. (I know we set it in the `fxci-config` decision tasks because it needs to be overwritten.)